<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sort</class>
 <widget class="QDialog" name="Sort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sort</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="firstGroupBox">
     <property name="title">
      <string>Первичный ключ</string>
     </property>
     <layout class="QGridLayout" name="gridLayout1">
      <item row="0" column="0">
       <widget class="QLabel" name="label1">
        <property name="text">
         <string>Столбец</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="firstColumnCombo">
        <item>
         <property name="text">
          <string>Пусто</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label2">
        <property name="text">
         <string>Порядок сортировки</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="firstOrderCombo">
        <item>
         <property name="text">
          <string>По возрастанию</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>По убыванию</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="secondGroupBox">
     <property name="title">
      <string>Второй ключ</string>
     </property>
     <layout class="QGridLayout" name="gridLayout2">
      <item row="0" column="0">
       <widget class="QLabel" name="label3">
        <property name="text">
         <string>Столбец</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="secondColumnCombo">
        <item>
         <property name="text">
          <string>Пусто</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label4">
        <property name="text">
         <string>Порядок сортировки</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="secondOrderCombo">
        <item>
         <property name="text">
          <string>По возрастанию</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>По убыванию</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="thirdGroupBox">
     <property name="title">
      <string>Третий ключ</string>
     </property>
     <layout class="QGridLayout" name="gridLayout3">
      <item row="0" column="0">
       <widget class="QLabel" name="label5">
        <property name="text">
         <string>Столбец</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="thirdColumnCombo">
        <item>
         <property name="text">
          <string>Пусто</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label6">
        <property name="text">
         <string>Порядок сортировки</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="thirdOrderCombo">
        <item>
         <property name="text">
          <string>По возрастанию</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>По убыванию</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="moreButton">
     <property name="text">
      <string>Больше</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonBoxLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Ok|QDialogButtonBox::Cancel</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
